jobsdb/jobsdb.go

```diff
@@ -747,8 +750,8 @@ func (jd *HandleT) workersAndAuxSetup() {
 	if jd.registerStatusHandler {
 		admin.RegisterStatusHandler(jd.tablePrefix+"-jobsdb", jd)
 	}
-
-	jd.BackupSettings = jd.getBackUpSettings()
+	jd.BackupSettings = &BackupSettingsT{}
```

Comment:
What does `sync.Once` in `BackupSettingsT` is supposed to protect us from happening twice? Here we are creating a new Settings value every time.

Reply:
Make sense, removed this from the struct completely, masterBackup key is now being taken care by single variable.